### ISSUE TITLE
Missing syntax highlighting for various guide articles

### ISSUE THREAD

**Issue by ojeytonwilliams** [freeCodeCamp/freeCodeCamp#35960]
**Describe your problem and how to reproduce it:**
Due to unsupported languages being used for code blocks, a lot of guide articles are not getting the appropriate highlighting.  This isn't a problem with content, just style - unfortunately they don't look as good as they could.

One way to see which languages are not working correctly is to build the client.  This will produce a bunch of warnings, one for each language it doesn't recognize:

> warning unable to find prism language 'sh' for highlighting. applying generic code block
warning unable to find prism language 'terminal' for highlighting. applying generic code block
warning unable to find prism language 'node' for highlighting. applying generic code block
warning unable to find prism language 'output' for highlighting. applying generic code block
warning unable to find prism language 'cs' for highlighting. applying generic code block
warning unable to find prism language 'c++' for highlighting. applying generic code block
warning unable to find prism language 'solidity' for highlighting. applying generic code block
warning unable to find prism language '#include<math.h>' for highlighting. applying generic code block
warning unable to find prism language '#include <math.h>' for highlighting. applying generic code block
warning unable to find prism language 'c#' for highlighting. applying generic code block
warning unable to find prism language 'cpp//empty operation in queue' for highlighting. applying generic code block
warning unable to find prism language 'xaml' for highlighting. applying generic code block
warning unable to find prism language 'code' for highlighting. applying generic code block
warning unable to find prism language 'javasctipt' for highlighting. applying generic code block
warning unable to find prism language 'syntax' for highlighting. applying generic code block
warning unable to find prism language 'unix' for highlighting. applying generic code block
warning unable to find prism language 'linux' for highlighting. applying generic code block
warning unable to find prism language '<!doctype html>' for highlighting. applying generic code block
warning unable to find prism language 'console' for highlighting. applying generic code block
warning unable to find prism language 'some examples of it are:' for highlighting. applying generic code block
warning unable to find prism language 'select query' for highlighting. applying generic code block
warning unable to find prism language 'java 8' for highlighting. applying generic code block
warning unable to find prism language 'algorithm' for highlighting. applying generic code block
warning unable to find prism language 'txt' for highlighting. applying generic code block
warning unable to find prism language 'swift4' for highlighting. applying generic code block
warning unable to find prism language 'react.js' for highlighting. applying generic code block
warning unable to find prism language 'javascriot' for highlighting. applying generic code block
warning unable to find prism language 'let quotesample = "the five boxing wizards jump quickly.";' for highlighting. applying generic code block
warning unable to find prism language 'javacsript' for highlighting. applying generic code block
warning unable to find prism language 'c++ -' for highlighting. applying generic code block

These all come from code blocks with languages that are not supported by Prism or are referred to differently.  For example, C++ is supported, but Prism only works if you use `cpp`. 

**Add a Link to the page with the problem:**
[This](https://www.freecodecamp.rocks/guide/algorithms/flood-fill) is a c++ example.  It currently looks like this:
![cppRocks](https://user-images.githubusercontent.com/15801806/57156182-9c5c3780-6ddd-11e9-9bc5-323d143392e4.png)
  It was probably intended to look like this:
![cpp](https://user-images.githubusercontent.com/15801806/57156101-720a7a00-6ddd-11e9-8d50-0fe37247befb.png)

**Tell us about your browser and operating system:**
* Browser Name: Chrome
* Browser Version: 74.0.3729.131 
* Operating System: Ubuntu

**Comment by RandellDawson** [comment]
I can work on this one.  This would be a simple replace across the guide for the various invalid language names.

**Comment by ojeytonwilliams** [comment]
It seems reasonably safe to assume the author meant `shell`.  I had a quick look and all the ones I saw were clearly intended to be that.

Maybe I'm being overly cavalier, but I think a global replace is bound to be an improvement.  Most of the time it's going to be right and it'll still be getting reviewed.

**Comment by RandellDawson** [comment]
I review everything and created a PR (#35961) which replaces all the ones where it was obvious where the replacements could be made globally.  However, there are a few (see below) which I was not quite sure what to do with:
````
```solidity - maybe just remove solidity because it is not supported?

```xaml - either remove xaml or changed it to xml which would be a similar syntax?

```algorithm - I think these should be handled in a separate PR because many are different in each case.  I would rather us be able to review these individually on a PR vs. trying to find them in the global replace I did in the PR mentioned above.
````

**Comment by ojeytonwilliams** [comment]
From what I [found](https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/xaml-overview-wpf), xaml is written using xml, so replacing that should be fine.

Algorithm looks to be fairly generic pseudocode, so I wouldn't highlight it.

Solidity is less clear.  We could leave it in, then it would highlight correctly if someone added it to Prism.  However, it would generate warnings when the site is built.  Not sure.

**Comment by RandellDawson** [comment]
I think replace `xaml` with `xml` should work.

I think leaving `solidity` off to deal get rid of the warning.

There are only 10 files using `Algorithm`.  Basically it is two files (the English plus 4 translations (Arabic not included).  The one file `merge-sort` file actually is pseudo-code, so we could just remove the `Algorithm` from those.  The other file `insertion-sort` actually is JavaScript code, so I think changing it to `js` is fine.

**Comment by moT01** [comment]
So is there still a number of files which need to be changed for this to get closed @ojeytonwilliams, @RandellDawson? We merged that large PR, but I think some files got missed from potential conflicts.

**Comment by RandellDawson** [comment]
@moT01 See [this comment](https://github.com/freeCodeCamp/freeCodeCamp/pull/35961#issuecomment-491458449).

**Comment by RandellDawson** [comment]
Based on [comments in the Contributors room](https://gitter.im/FreeCodeCamp/Contributors?at=5d0c0626e7267a62a90f75e2), I am reopening this as an ongoing issue.

I will create a new PR to address the most recent warnings.

**Comment by ojeytonwilliams** [comment]
Fixed by #36318
